config/config.yaml

```diff
@@ -0,0 +1,157 @@
+defaults:
+  region: {{ .ctx.region }}
+  # Resourcegroups
+  globalRG: global
+  regionRG: hcp-underlay-{{ .ctx.region }}-{{ .ctx.regionStamp }}
+  serviceClusterRG: hcp-underlay-{{ .ctx.region }}-{{ .ctx.regionStamp }}-svc
+  managementClusterRG: hcp-underlay-{{ .ctx.region }}-{{ .ctx.regionStamp }}-mgmt-{{ .ctx.cxStamp }}
+  imageSyncRG: hcp-underlay-{{ .ctx.region }}-{{ .ctx.regionStamp }}-imagesync
+
+  # General AKS config
+  kubernetesVersion: 1.30.5
+  istioVersion: "['asm-1-22']"
+  vnetAddressPrefix: "10.128.0.0/14"
+  persist: true
```

Comment:
Called it out above, but would be nice to not persist by default, and instead make it on the user to do this.  

For dev-infra that's persistent, set it on github actions workflows.

Reply:
https://github.com/Azure/ARO-HCP/pull/762/commits/19cd76d22e8e5d6dc6130818a33068ea37b98e6a